includes/class.llms.playnice.php

```diff
@@ -52,11 +53,45 @@ public function __construct() {
 		// WPEngine heartbeat fix.
 		add_filter( 'wpe_heartbeat_allowed_pages', array( $this, 'wpe_heartbeat_allowed_pages' ) );
 
+		// BuddyBoss profile nav compatibility issue fix (the nav is set up at priority 6).
+		add_action( 'bp_init', array( $this, 'buddyboss_compatibilty' ), 5 );
+
 		// Load other playnice things based on the presence of other plugins.
 		add_action( 'init', array( $this, 'plugins_loaded' ), 11 );
 
 	}
 
+	/**
+	 * Compatibility for BuddyBoss.
+	 *
+	 * @since [version]
+	 *
+	 * @link https://github.com/gocodebox/lifterlms/issues/2142#issuecomment-1157924080.
+	 *
+	 * @return void
+	 */
+	public function buddyboss_compatibilty() {
+
+		if ( ! function_exists( 'is_plugin_active' ) || ! function_exists( 'bp_is_my_profile' ) || bp_is_my_profile() ) {
+			return;
+		}
+
+		if ( is_plugin_active( 'buddyboss-platform/bp-loader.php' ) ||
+				( is_multisite() && is_plugin_active_for_network( 'buddyboss-platform/bp-loader.php' ) ) ) {
```

Comment:
Readability. I thought there was a standard about multi-line 'if' statements separating the conditions from the "if" parenthesis, but I couldn't find a reference to one. Maybe I'm thinking of multi-line function arguments?

Anyway, how about this?
```suggestion
		if (
			is_plugin_active( 'buddyboss-platform/bp-loader.php' ) ||
			( is_multisite() && is_plugin_active_for_network( 'buddyboss-platform/bp-loader.php' ) )
		) {
```

Reply:
looks goood